readyapi4j-groovy-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/pro/DataGenDataSourceTestStepDslTest.groovy

```diff
@@ -0,0 +1,186 @@
+package com.smartbear.readyapi4j.dsl.pro
+
+import com.smartbear.readyapi.client.model.BooleanDataGenerator
+import com.smartbear.readyapi.client.model.BooleanDataGenerator.FormatEnum
+import com.smartbear.readyapi.client.model.ComputerAddressDataGenerator
+import com.smartbear.readyapi.client.model.CustomStringDataGenerator
+import com.smartbear.readyapi.client.model.DataGenDataSource
+import com.smartbear.readyapi.client.model.DataGenerator
+import com.smartbear.readyapi.client.model.DataSourceTestStep
+import com.smartbear.readyapi.client.model.DateAndTimeDataGenerator
+import com.smartbear.readyapi.client.model.StateNameDataGenerator
+import com.smartbear.readyapi.client.model.StringDataGenerator
+import com.smartbear.readyapi4j.TestRecipe
+import org.junit.Test
+
+import static com.smartbear.readyapi.client.model.BooleanDataGenerator.FormatEnum.TRUE_FALSE
+import static com.smartbear.readyapi.client.model.BooleanDataGenerator.FormatEnum.YES_NO
+import static com.smartbear.readyapi.client.model.BooleanDataGenerator.FormatEnum._1_0
+import static com.smartbear.readyapi.client.model.ComputerAddressDataGenerator.AddressTypeEnum.IPV4
+import static com.smartbear.readyapi.client.model.ComputerAddressDataGenerator.AddressTypeEnum.MAC48
+import static com.smartbear.readyapi.client.model.DateAndTimeDataGenerator.DateTimeFormatEnum.YYYY_MM_DDTHH_MM_SSZ_ISO_8601_
+import static com.smartbear.readyapi4j.dsl.ServerTestDsl.recipe
+
+class DataGenDataSourceTestStepDslTest {
+    @Test
+    void createsRecipeWithDataSourceTestStepWithBasicDataGenerators() throws Exception {
+        TestRecipe testRecipe = recipe {
+            dataGenDataSource 'CityDataGenDataSource', {
+                numberOfRows 2
+                cityNames 'cityNames'
```

Comment:
I wonder if the plural 'cityNames' may be a bit confusing. I think that I would prefer 'cityName', although this will of course generate one name per row.

The same goes for the other methods in the datagen part of the DSL - countryNames, streetAddresses, yesNoBooleanValues etc.